ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -290,14 +324,19 @@ function WorkloadCvesOverviewPage() {
                     component="div"
                     className="pf-v5-u-pl-lg pf-v5-u-background-color-100"
                 >
-                    <VulnerabilityStateTabs onChange={() => pagination.setPage(1)} />
+                    <VulnerabilityStateTabs
+                        onChange={() => {
+                            setObservedCveMode('WITH_CVES');
+                            pagination.setPage(1, 'replace');
```

Comment:
The "No CVEs" view is only supported for Observed - so if these tabs are changed we need to reset to "With CVEs" mode.